modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/ScalaClientCodegen.java

```diff
@@ -75,8 +80,8 @@ public ScalaClientCodegen() {
         importMapping.remove("Set");
         importMapping.remove("Map");
 
-        importMapping.put("DateTime", "org.joda.time.DateTime");
-        importMapping.put("ListBuffer", "scala.collection.mutable.ListBuffer");
+        importMapping.put("Date", "java.util.Date");
+        importMapping.put("ListBuffer", "scala.collections.mutable.ListBuffer");
```

Comment:
This change was probably missed during a merge conflict. It should be `scala.collection.mutable.ListBuffer`